save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/Requests.kt

```diff
@@ -96,6 +96,28 @@ internal suspend fun SaveAgent.downloadAdditionalResources(
         }
 }
 
+/**
+ * Downloads binary of save-cli into the current directory
+ *
+ * @param url
+ * @throws IllegalStateException
+ */
+internal suspend fun SaveAgent.downloadSaveCli(url: String) {
```

Comment:
where we download save-cli from? Shouldn't we hardcode github URL here?

Reply:
No, it's configurable from agent settings and also we are downloading it from backend, where it is put to during build. In the future we can store it in S3 to be independant from remote services